transport/http/handler.go

```diff
@@ -203,15 +213,16 @@ func handleOnewayRequest(
 			Handler: onewayHandler,
 			Logger:  logger,
 		})
-		updateSpanWithErr(span, err)
+		updateSpanWithErr(span, err, yarpcerrors.FromError(err).Code())
 	}()
 	return nil
 }
 
-func updateSpanWithErr(span opentracing.Span, err error) {
+func updateSpanWithErr(span opentracing.Span, err error, errCode yarpcerrors.Code) {
```

Comment:
Let's add some consistency between protocols:
- Name it "updateSpanWithErrAndCode"
- Add "error" as returned argument
- Replace "update.. + return err" with "return update...(...err...)"

Reply:
Absolutely, this PR was implemented in parallel with yarpc/grpc. I will update it on top of yarpc/grpc (#2282) to avoid conflicts.